File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        int res=0;
        unordered_set<int> store(nums.begin(),nums.end());

        for(int num:nums){
            int cur=num,streak=0;
            while(store.find(cur)!=store.end()){
                streak++;
                cur++;
            }
            res=max(res,streak);
        }
        return res;
    }
};

File: Data Structures & Algorithms/duplicate-integer/submission-1.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        dict={}
        for n in nums:
            if n in dict:
                dict[n]+=1
            else:
                dict[n]=1
        for i in dict:
            if dict[i]>=2:
                return True
        return False

         